.metalinter.json

```diff
@@ -14,8 +14,7 @@
     }
   },
   "Enable":
-    [ "deadcode"
-    , "varcheck"
+    [ "varcheck"
```

Comment:
should revert this for now, and open an issue to remove all references to `nolint: deadcode` too

Reply:
handled in another PR